### PR TITLE
increase perf timeout for running more tests

### PR DESCRIPTION
**Pull request by carolynhu**
This PR can only be merged after this: https://github.com/istio/tools/pull/591

### CODE REVIEW

File: prow/cluster/jobs/all-periodics.yaml [hunk 1]
```diff
@@ -209,7 +209,32 @@ periodics:
...
         command:
           - entrypoint
-          - perf/benchmark/run_benchmark_job.sh
+          - perf/benchmark/run_istio_benchmark_job.sh
+
+- cron: "0 8 * * *" # starts every day at 08:00AM UTC
+  name: daily-linkerd-performance-benchmark
+  branches: master
+  decorate: true
+  timeout: 12h
```
Comment (howardjohn) [inline prow/cluster/jobs/all-periodics.yaml]: does this one take 12hr as well?

Comment (carolynhu) [reply prow/cluster/jobs/all-periodics.yaml]: it should be like around 3 hr actually. I just put it here in case we have more tests configurations.

Comment (howardjohn) [reply prow/cluster/jobs/all-periodics.yaml]: How does it become 3hrs? it looks like the test is just running at 6 different# of connections for 4min each shouldn't that be 24min?

Comment (carolynhu) [reply prow/cluster/jobs/all-periodics.yaml]: sorry, it should be  4min * 6 * 3= 72 min

**Review by mandarjog — APPROVED** [review]
We should just have captive clusters now.
12 hours is already a lot, and we want to eliminate variation between tests.

**Comment by mandarjog** [comment]
john is it ok to increase timeouts to these levels? is there a scheduling issue here with successive tests?

**Review by howardjohn — APPROVED** [review]
Should be fine. We have two clusters for this, so as long as the tests don't go over 24hrs (realistically 20hrs or so to give us a buffer) we should be fine? SEe https://velodrome.istio.io/dashboard/db/boskos-dashboard?orgId=1&from=now-30d&to=now the bottom panel

**Pull request merged**